src/config-schemas/external-patient-chart-config.ts

```diff
@@ -0,0 +1 @@
+export interface ExternalPatientChartConfig {}
```

Comment:
What is this, why does it exist?

Reply:
It was to act as a contract for configs in 3rd party patient chart widgets module. It might be that you might want to enforce some structure in these configs if we are going to help load them using the configs. I don't have strong opinion about it, and might as well get rid of it.

Reply:
Ah, I understand now! That's a great idea, and is something we should definitely have eventually. But I think we should wait to figure out where it should go and how it should look until we have some concrete requirement for it not to be empty.